### DISCUSSION TITLE
添加热点分享功能

### DISCUSSION THREAD

**Discussion by linuxblog996** [SagerNet/SagerNet#84] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
我想把一个代理通过热点的方式分享给其他设备，使其不用打开代理也可以访问网络

**Describe the solution you'd like**
添加热点分享功能

**Comment by nekohasekai** [comment, marked as answer]
用 https://github.com/Mygod/VPNHotspot

**Comment by linuxblog996** [comment]
好的，谢谢你
